ACCS-366: My Account: Stored payment methods page (#816)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/user-account/dictionary.mdx

For multiline fields like street address, each line has its own field code. The first line uses the base field code (for example, `street`), and subsequent lines append `_multiline_N` where _N_ is the line number (for example, `street_multiline_2` for the second line).
</Aside>

## Stored payment methods (Payments)

> The **PaymentMethods** container and **PaymentCard** component use keys under **Account.Payments** for the section title, error messages, and per-card labels. Empty states when there are no saved methods reuse **EmptyList.Payments** under both **Account.minifiedView** and **Account.fullSizeView** (same message in each view by default).

Reviewer: ```suggestion
The PaymentMethods container and PaymentCard component use keys under Account.Payments for the section title, error messages, and per-card labels. When no saved methods exist, the empty state uses `EmptyList.Payments` in both `Account.minifiedView` and `Account.fullSizeView`, with the same default message in each view.
```
